<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Adm_user_manage</class>
 <widget class="QWidget" name="Adm_user_manage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>1081</width>
     <height>641</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>工号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>姓名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>性别</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>部门</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>职位</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>住址</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>出生日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>手机号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>邮箱</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>密码</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>任职状态</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btn_back_admin">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>600</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>添加一个</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>请选择</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>姓名</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>工号</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>出生日期</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_delete">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>470</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_quit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>690</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qurry_1">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="qurry_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>请选择</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>姓名</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>工号</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>部门</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>职位</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>姓名模糊查询</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>工号模糊查询</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>职位+部门</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>姓名+职位</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>姓名+部门</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>手机号+邮箱模糊</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>姓名+地址模糊</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>批量添加</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="path_from">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>文件路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>员工信息录入：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
